test/e2e/delete_test.go

```diff
@@ -81,12 +81,12 @@ func (tc *testContext) testBYOHRemoval(t *testing.T) {
 // checkDirsDoNotExist returns true if the required directories do not exist on the Windows instance with the given
 // address
 func (tc *testContext) checkDirsDoNotExist(address string) (bool, error) {
-	command := remotePowerShellCmdPrefix
```

Comment:
You've removed all usages of `remotePowerShellCmdPrefix` but not [the constant itself](https://github.com/openshift/windows-machine-config-operator/pull/663/files#diff-cd50fad6da0839c40258ca356bf995a7ee5e1d5c270ee0af2cc5752e2f10a454R25)

Reply:
Thanks, I forgot this constant existed. Using this as it is already in-use, instead of exporting the one from the `Windows` package.